composer.json

```diff
@@ -115,7 +115,6 @@
         "drupal/restui": "^1.21",
         "drupal/search_api": "^1.31",
         "drupal/twig_tweak": "^3.2.1",
-        "drupal/twigsuggest": "^1.0@RC",
```

Comment:
Maybe the old commit enabling this could be removed.

Reply:
Yeah, but I think it still has some value in documenting the process to [`novel_theme_suggestions_alter()` in case someone comes along looking for an alternate at some point](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/574600bda1fac037093222f816870bffc6b475bd/web/themes/custom/novel/novel.theme#L234-L234).